.github/workflows/docker.yml

```diff
@@ -141,3 +144,8 @@ jobs:
           ${{ github.event_name == 'release' && format('--tag {0}/{1}:stable', env.CONTAINER_NAMESPACE, matrix.name) || '' }}
           ${{ github.event_name == 'release' && format('--tag {0}/{1}:{2}', env.CONTAINER_NAMESPACE, matrix.name, github.ref_name) || '' }}
           ${{ inputs.image-tag && format('--tag {0}/{1}:{2}', env.CONTAINER_NAMESPACE, matrix.name, inputs.image-tag) || '' }}
+
+      - name: Log out of Docker and disable Docker Notary
+        run: |
+          docker logout
+          echo "DOCKER_CONTENT_TRUST=0" >> $GITHUB_ENV
```

Comment:
Does this part matter, considering the agents don't retain their state between runs?

Reply:
In this particular case it doesn't matter. But we tend to include it for future if more steps will be added after this 😃

Reply:
Makes sense. Do you think this can also be distributed as a bitwarden action then, to make things cleaner? 🙂

Reply:
It can be distributed as a bitwarden action. But it's so small that I don't think it makes sense to make them as reusable action.